tensorflow/core/kernels/maxpooling_op.cc

```diff
@@ -1291,7 +1325,7 @@ template <typename T>
 struct LaunchMaxPoolingWithArgmax<Eigen::GpuDevice, T> {
   static void launch(OpKernelContext* context, const PoolParameters& params,
                      const Tensor& input, Tensor* output, Tensor* argmax,
-                     bool propagate_nans) {
+                     bool propagate_nans, bool include_batch_in_index) {
     bool status = functor::MaxPoolForwardWithOptionalArgmax<T>()(
```

Comment:
Fail a `CHECK` or raise NotImplemented if `include_batch_in_index` is True. And same for `LaunchMaxPoolingGradWithArgmax` and `LaunchMaxPoolingGradGradWithArgmax`